Sources/libhostmgr/Platforms/VMManager.swift

```diff
@@ -38,7 +38,7 @@ public struct VMManager {
 
     /// Delete a local VM
     ///
-    public func removeVM(name: String) async throws {
+    public static func removeVM(name: String) async throws {
```

Comment:
Find to keep this an instance method but I try to go stateless when possible, and it helped us avoid a long lived VMManager.